admin/src/Components/ProductForm.jsx

```diff
@@ -116,7 +117,12 @@ class ProductForm extends React.Component {
                     </fieldset>
                     <fieldset className="uk-margin-top">
                         <legend><i className="uk-icon-filter"/> Filter</legend>
-                        <SelectInput model={product} name="filter" title="Filter" getLabel={o => o.name} getValue={o => o.id} options={[{id: "", name: "No filter"}, {id: "start_package", name: "Startpaket"}]}/>
+                        <SelectInput model={product} name="filter" title="Filter" getLabel={o => o.name} getValue={o => o.id} options={[
+                            {id: "", name: "No filter"},
+                            {id: "start_package", name: "Startpaket"},
+                            {id: "labaccess_non_subscription_purchase", name: "Labaccess Non Subscription Purchase"},
+                            {id: "membership_non_subscription_purchase", name: "Membership Non Subscription Purchase"},
+                        ]}/>
```

Comment:
I feel like the filter needs some explanation for the user (i.e. for an administrator). I.e. what it does depending on its value. It would be useful with a tooltip on the filter, and a more descriptive name for the different alternatives.

Reply:
Fixed in 943e5e44